replica/server.go

```diff
@@ -102,6 +104,20 @@ func (s *Server) Create(size int64) error {
 	return r.Close()
 }
 
+func (s *Server) Preload() error {
+	if s.r == nil {
+		return fmt.Errorf("Can't read extents, s.r is nil")
+	}
+
+	s.PreloadStatus = "Started"
```

Comment:
To avoid multiple of these go routines to get started, PreloadStatus need to be set to 'Started' even before this go routine is created. In case if you want to know that go-routine is started, you can have some other state.